src/Tribe/Attendees.php

```diff
@@ -553,6 +553,49 @@ private function generate_filtered_list( $event_id ) {
 		return array_filter( $rows );
 	}
 
+	/**
+	 * Sanitize rows for CSV usage.
+	 *
+	 * @since 4.10.7.2
+	 *
+	 * @param array $rows Rows to be sanitized.
+	 *
+	 * @return array Sanitized rows.
+	 */
+	public function sanitize_csv_rows( array $rows ) {
+		foreach ( $rows as &$row ) {
+			$row = array_map( [ $this, 'sanitize_csv_value' ], $row );
+		}
+
+		return $rows;
+	}
+
+	/**
+	 * Sanitize a value for CSV usage.
+	 *
+	 * @since 4.10.7.2
+	 *
+	 * @param mixed $value Value to be sanitized.
+	 *
+	 * @return string Sanitized value.
+	 */
+	public function sanitize_csv_value( $value ) {
+		if (
+			0 === mb_strpos( $value, '=' )
+			|| 0 === mb_strpos( $value, '+' )
+			|| 0 === mb_strpos( $value, '-' )
+			|| 0 === mb_strpos( $value, '@' )
```

Comment:
Maybe add pipe?

Reply:
This only covers starting characters, but specifically these characters that we're looking for: https://www.owasp.org/index.php/CSV_Injection